### PR TITLE
Math magicians: Complete website

### PR DESCRIPTION
**Pull request by KingsleyIbe**
# Milestone Changes
### In this milestone, I implemented the following:

- Created a website consisting of 3 pages: Home, Calculator, and Quote.
- Maintained a general layout similar to the provided wireframes in the project requirement.
- Added my own styles to improve the look and feel of the project using React basic styling.
- For every page, you should create a route. Created a route for every page.
- Ensured that navigation links for all 3 routes are displayed on each page.
- Used React-Router to make the routes work.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,10 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import Calculator from './Components/Calculator';
+import Router from './Components/Router';
+// import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+// import Calculator from './Components/Calculator';
+// import Header from './Components/Header';
```
Comment (bimbolabuari) [inline src/index.js]: Kindly remove all commented code ✊ 

Please check for other occurrences 🙂

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @KingsleyIbe, 

You have done a good job so far 🚀  

## Highlights 🎉 

- Nice UI design
- React basic styling
- Navigation links for all 3 routes are displayed on each page.
- Use React Router to make the routes work.

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

**Comment by KingsleyIbe** [comment]
Hey Abimbola,

Thank you for your timely feedback, be sure changes will be implemented.

**Review by DanSam5K — APPROVED** [review]
## Hi @KingsleyIbe, ✌️ Approved✌️ 
- [x] Excellent job implementing all the changes suggested by the previous reviewer ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KingsleyIbe** [comment]
Hey Daniel,

Thanks for your prompt response and kind approval.

**Comment by DanSam5K** [comment]
You welcome @KingsleyIbe

**Pull request merged**